### ISSUE TITLE
Updating dependencies will cause compilation to fail

### ISSUE THREAD

**Issue by mosajjal** [folbricht/routedns#421]
Hi,

if you run `go get -v -u ./...` on the repo, it upgrades `github.com/cisco/go-hpke` to the latest version. This causes `github.com/cloudflare/odoh-go` to fail. since `odoh-go` is archived by CF, I don't hold any hope they will fix this. Just something to keep in mind long-term

![image](https://github.com/user-attachments/assets/69d76bbe-cb2d-430c-8c66-7f1c3dc9cd68)

**Comment by folbricht** [comment]
@LeonardWalter Is this something we could work around maybe to keep supporting odoh?

**Comment by LeonardWalter** [comment]
Yes, we could move to https://github.com/sthorne/odoh-go. It is a forked version that fixes the SIKE issue.

**Comment by folbricht** [comment]
Thank you. Just merged https://github.com/folbricht/routedns/pull/423, hope that fixes the issue

**Comment by mosajjal** [comment]
yep it looks good now. thanks!
